mytrip/src/main/java/pl/mytrip/trip/Controllers/TripController.java

```diff
@@ -46,4 +46,8 @@ TripDTO updateTrip(@RequestBody TripDTO dto, @PathVariable String id) {
     void deleteTrip(@PathVariable String id) {
         tripService.deleteTrip(id);
     }
+
+    @RequestMapping(value = "/login", method = RequestMethod.GET)
```

Comment:
a to url w tym wypadku nie będzie wyglądał tak: /api/trips/login?

Reply:
będzie, ale się nie przejmowałem tym